features/verblijfstitel.feature

```diff
@@ -42,22 +42,17 @@ Functionaliteit: Verblijfstitel
 
     @gba
     Abstract Scenario: verblijfstitel heeft <titel>
-      Gegeven het systeem heeft een persoon met de volgende gegevens
-      | naam                | waarde    |
-      | burgerservicenummer | 555550002 |
-      En de persoon heeft de volgende 'verblijfstitel' gegevens
-      | naam                                | waarde       |
-      | Aanduiding verblijfstitel (39.10)   | 37           |
-      | Datum einde verblijfstitel (39.20)  | <datumEinde> |
-      | Ingangsdatum verblijfstitel (39.30) | 20210315     |
-      Als personen op '5 juni 2022' wordt gezocht met de volgende parameters
```

Comment:
Om dit te realiseren moet de automation code de gba container stoppen, systeem tijd aanpassen en weer opstarten. Dit lijkt mij niet handig.
Wat wel kan om datum placeholders te gebruiken. Destijds heb ik aangegeven dat 'vandaag' als waarde voor datumEinde kan worden opgegeven. Dan zorgt de automation code dat dit wordt vertaald naar de datum van vandaag. Voor een datum in het verleden kan 'gisteren' worden gebruikt als datum placeholder. Ik weet alleen niet of dat voor jullie een acceptabele oplossing is

Reply:
lijkt mij prima.